hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/info/NodeInfo.java

```diff
@@ -58,4 +60,29 @@ public interface NodeInfo {
      * @return this node's account memo
      */
     String memo();
+
+    /**
+     * The host name of this node, as known by the external world. This is an IP address.
+     *
+     * @return The host name (IP Address) of this node
+     */
+    String externalHostName();
```

Comment:
If this is an IP address, would it be better to return an INetAddress?

Reply:
I wondered, or at least a better name. This is the name used by platform FWIW.